Doc/library/sqlite3.rst

```diff
@@ -640,6 +640,11 @@ Cursor Objects
       .. versionchanged:: 3.6
          Added support for the ``REPLACE`` statement.
 
+   .. attribute:: arraysize
+
+      This attribute allows you to set the number of rows returned by :meth:`fetchmany`.
+      The default value is 1 meaning a single row would be fetched per call.
```

Comment:
Yeah, our docs aren't fully consistent, especially the older docs.  So when we touch them we sometimes update the style...or sometimes we follow the existing style depending on what the updater feels is best.  In this case the style appears to be a mix, so I vote for moving it toward our current style.  

I was hoping that 'controls' coupled with 'default' would make it clear you could set it, but maybe that isn't enough?